### ISSUE TITLE
Inter character rendering issue when exporting GSlides to PDF

### ISSUE THREAD

**Issue by thejulianw** [google/fonts#2425]
Using **Inter** in Google Slides and exporting the deck to PDF renders the capital A character wrongly when viewing the PDF in Preview (macOS). Opening it in Google Drive's preview works fine.
This is said to be an issue on Googles side, since the font works flawlessly everywhere else.

<img width="221" alt="EVuFIUrWoAA-RiS" src="https://user-images.githubusercontent.com/63803032/79476135-e3503680-8008-11ea-8e6d-8a0cd8993933.png">

Thanks,

Julian

**Comment by sladen** [comment]
Julian: Root cause is handling of overlaps.  However to allow please provide the exact PDF.   (Or a self-contained example PDF also exhibiting the problem).

**Comment by thejulianw** [comment]
Hi sladen,

thank you, I attached an example PDF and a screenshot of how it looks on my Mac when opened in Preview.
The PDF was created by building a slide in Google Slides, then exported to PDF by going "File" -> "Download" -> "PDF Document (.pdf)"

Some of my colleagues have the same issue, whereas others report it looks normal on their machine (all running latest macOS).

<img width="946" alt="InterFont_GoogleSlidesExport_Example_Screenshot" src="https://user-images.githubusercontent.com/63803032/79479913-bce0ca00-800d-11ea-9141-96ba66f3bf27.png">

[InterFont_GoogleSlidesExport_Example.pdf](https://github.com/google/fonts/files/4488444/InterFont_GoogleSlidesExport_Example.pdf)

**Comment by thlinard** [comment]
The PDF looks fine in Adobe Acrobat, but wrong in Apple Preview.

**Comment by thejulianw** [comment]
> The PDF looks fine in Adobe Acrobat, but wrong in Apple Preview (tested with Preview v11, macOS 10.15.4).

That is true, it also looks fine when viewing in Google Drive for instance.
Some of my colleagues even say it looks good in Preview for them, however, another colleague and I can't get it to work for us in macOS Preview.

**Comment by lindig** [comment]
The A most likely is not the only issue. See the outlines below.

![inter-open-sores](https://user-images.githubusercontent.com/843617/84367126-b1ec8500-abcb-11ea-8e08-cff572d6bd83.png)

**Comment by laurisbernhart** [comment]
We are also facing this problem with **Comfortaa**, when creating a PDF with chromium. Adobe works fine but in Apple preview, the A has these two dots. 

Using Comfortaa from eg fontsquirrel - doesn't come with this issue.

**Comment by lindig** [comment]
This could be an issue by the distribution via Google Fonts which are compiled from source code by Goole, I assume. The developer of the font suggests to download the fonts from his GitHub repository.
